lm_eval/__main__.py

```diff
@@ -301,7 +310,7 @@ def cli_evaluate(args: Union[argparse.Namespace, None] = None) -> None:
 
     if args.include_path is not None:
         eval_logger.info(f"Including path: {args.include_path}")
-    task_manager = TaskManager(args.verbosity, include_path=args.include_path)
+    task_manager = TaskManager(include_path=args.include_path)
```

Comment:
This changes the signature of TaskManager init--we need to maintain backward compatibility here unfortunately!